<ui version="4.0" >
 <class>GraphEditWidget</class>
 <widget class="QWidget" name="GraphEditWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>624</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="1" >
          <widget class="KRuler" name="_horizontalRuler" >
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="showTinyMarks" >
            <bool>true</bool>
           </property>
           <property name="length" >
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="KRuler" name="_verticalRuler" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QGraphicsView" name="_graphicsView" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QDoubleSpinBox" name="_zoomLevelSpinBox" >
           <property name="maximum" >
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KSqueezedTextLabel" name="_txtStatus" >
           <property name="text" >
            <string>KSqueezedTextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KComboBox" name="_cmbEngine" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text" >
             <string>javascript</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>python</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>ruby</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btnRunScript" >
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" native="1" name="_mutantWidget" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="KTabWidget" name="tabWidget" >
         <property name="tabPosition" >
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="_tabEditor" >
          <attribute name="title" >
           <string>Editor</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="_tabDebug" >
          <attribute name="title" >
           <string>Debug</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="KTextBrowser" name="_txtDebug" />
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <widget class="QPushButton" name="_btnClearDebugArea" >
               <property name="text" >
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KRuler</class>
   <extends>QWidget</extends>
   <header>kruler.h</header>
  </customwidget>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_btnClearDebugArea</sender>
   <signal>clicked()</signal>
   <receiver>_txtDebug</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>53</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
